fetch-publication/src/main/java/no/unit/nva/publication/fetch/FetchPublicationHandler.java

```diff
@@ -31,44 +32,42 @@
     @JacocoGenerated
     public FetchPublicationHandler() {
         this(new DynamoDBPublicationService(
-                        AmazonDynamoDBClientBuilder.defaultClient(),
-                        objectMapper,
-                        new Environment()),
-                new Environment());
+                AmazonDynamoDBClientBuilder.defaultClient(),
+                objectMapper,
+                new Environment()),
+            new Environment());
     }
 
     /**
      * Constructor for MainHandler.
      *
-     * @param publicationService    publicationService
-     * @param environment  environment
+     * @param publicationService publicationService
+     * @param environment        environment
      */
     public FetchPublicationHandler(PublicationService publicationService,
                                    Environment environment) {
         super(Void.class, environment);
```

Comment:
Should we send the logger into super constructor?

Reply:
Done